plugins/generator-1.20.4/neoforge-1.20.4/templates/particle.java.ftl

```diff
@@ -101,6 +101,15 @@ package ${package}.client.particle;
 		return ParticleRenderType.PARTICLE_SHEET_${data.renderType};
 	}
 
+
+	<#if hasProcedure(data.scale)>
+	@Override public float getQuadSize(float scale) {
+		float k = ((float) this.age + scale) / (float) this.lifetime;

```

Comment:
Why all this math?

Reply:
it is used for a gradual descent or increase in the size of the particle, but I am still working on it since as I said, in my opinion it would be useful in addition to the procedure, the initial size of the particle

Reply:
But this is something user can do in procedure. You are now altering what this PR does...

Please rever this back to what PR was intended to do...

Reply:
Ok

Reply:
But what would the super method do if I called it?

Reply:
Return

![image](https://github.com/MCreator/MCreator/assets/16374228/18689018-4d2a-4b81-a0d4-88c1f6fa83c0)

but implemented more correctly and universally in case this method in parent class changes in the future

Reply:
Done!